### PR TITLE
Display current and desired replicas on overview page

### LINKED ISSUE #2584: Wanted: show replica count / current number of pods in deployment view
This view should show the current replica count for the deployed RC and pod count.

http://www.screencast.com/t/sIG8C83OP0

### PR DESCRIPTION
**Pull request by spadgett**
openshift_web_console

screenshot_6_5_15__4_19_pm

Fixes #2584

### CODE REVIEW

File: assets/app/styles/_components.less [hunk 1]
```diff
@@ -52,6 +52,9 @@
       + span {
         margin-left: 5px;
       }
+      .component-status {
```
Comment (liggitt) [inline assets/app/styles/_components.less]: we'll also probably use this when we inline the service name with the component label... maybe call it `component-label-details`?

File: assets/app/views/_overview-deployment.html [hunk 2]
```diff
@@ -76,13 +76,18 @@ <h3 class="h4">{{rc.metadata.name}}</h3>
     <div class="component-label">Pod Template</div>
 
     <pod-template
-      pod-template="rc.spec.template" 
+      pod-template="rc.spec.template"
       images-by-docker-reference="imagesByDockerReference"
       builds="builds">
     </pod-template>
 
     <!-- TODO: surface spec.replicas/status.replicas here? -->
```
Comment (liggitt) [inline assets/app/views/_overview-deployment.html]: remove TODO :)

File: assets/app/views/_overview-deployment.html [hunk 3]
```diff
@@ -76,13 +76,18 @@ <h3 class="h4">{{rc.metadata.name}}</h3>
...
-    <div class="component-label">Pods</div>
+    <div class="component-label">
+      Pods
+      <span ng-if="rc.status.replicas === rc.spec.replicas" class="component-status">({{rc.status.replicas}})</span>
+      <span ng-if="rc.status.replicas !== rc.spec.replicas" class="component-status">({{rc.status.replicas}} current, {{rc.spec.replicas}} desired)</span>
```
Comment (liggitt) [inline assets/app/views/_overview-deployment.html]: Check behavior when you scale up a lot, then set spec.replicas to 0. If I'm reading the API correctly, when spec.replicas is "0" it gets omitted from the API, when status.replicas is "0", it gets included. Don't want to show `X current, desired`

**Comment by spadgett** [comment]
screen_shot_2015-06-06_at_9_54_40_am

**Comment by liggitt** [comment]
LGTM, [merge]

I'm curious about the gap in the last screenshot

**Comment by spadgett** [comment]
@liggitt The gap is from an animation. A pod was disappearing.

**Pull request merged**